src/Database/NeoServer.Data/Configurations/WorldEntityConfiguration.cs

```diff
@@ -15,6 +15,23 @@ public void Configure(EntityTypeBuilder<WorldEntity> builder)
         builder.Property(e => e.Name).IsRequired();
         builder.Property(e => e.Ip).IsRequired();
         builder.Property(e => e.Port).IsRequired();
+        builder.Property(e => e.MaxCapacity).HasDefaultValue(100).IsRequired();
+        builder.Property(w => w.RequiresPremium).IsRequired();
+        builder.Property(w => w.TransferEnabled).IsRequired();
+        builder.Property(w => w.AntiCheatEnabled).IsRequired();
+        builder.Property(w => w.CreatedAt).IsRequired();
+
+        builder.Property(w => w.Continent)
+            .HasConversion<string>()
+            .HasMaxLength(50);
+
+        builder.Property(w => w.PvpType)
+            .HasConversion<string>()
+            .HasMaxLength(50);
+
+        builder.Property(w => w.Mode)
```

Comment:
I think "WorldType" is more meaningful then just "Mode"

Reply:
Humm, I disagree; the world type is redundant. I believe mode is a good option.